carlos/rocker/nosotros.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Tech Blog</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans|PT+Sans:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+    <header class="site-header">
+        <div class="contenedor">
+            <div class="barra">
+                <a href="/">
+                    <h1 class="no-margin">Tech<span> Blog</span></h1>
+                </a>
+                <nav class="navegacion">
+                    <a href="nosotros.html">About Us</a>
+                    <a href="cursos.htmnl">Vision</a>
+                    <a href="contacto.html">Contact</a>
+                </nav>
+            </div>
+            <!--barra---->
+            <div class="texto-header">
+                <h2 class="no-margin"><span class="back">News about new techs</span></h2>
+                <p class="no-margin"><span class="back">Stay tunned!</span></p>
+            </div>
+        </div>
+        <!--contendedor-->
+    </header>
+
+    <main class="contenedor">
+        <h2 class="centrar-texto">About Us</h2>
+        <div class="grid">
+            <div class="columnas-6">
+                <img src="img/nosotros.jpg" alt="Imagen Nosotros">
+            </div>
+            <div class="columnas-6">
+                <p>Integer ipsum massa, suscipit a purus ac, dictum varius turpis. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Phasellus at pretium enim. Aliquam sit amet risus imperdiet, eleifend elit non, efficitur
+                    tellus. Quisque sit amet dignissim lectus, quis mollis tellus. Quisque finibus purus eget malesuada gravida.</p>
+                <p>Cras placerat dui ut commodo gravida. Nunc facilisis sapien sed tortor pharetra convallis. Fusce a ante dolor. Phasellus vestibulum auctor quam, vitae convallis tortor. Maecenas eget turpis lobortis, aliquet felis at, malesuada lorem.</p>
+            </div>
+        </div>
+    </main>
+
+    <footer class="site-footer">
+        <div class="contenedor">
+            <div class="barra">
+                <p class="no-margin">Blog <span>DeCafé</span></p>
```

Comment:
Here this section looks better than in the index.html, but why the text changed to `Blog deCafe`?